src/pages/components/split-button.mdx

```diff
@@ -1,10 +1,89 @@
 ---
 title: Split Button
 description: >
-  Description will be here
+  A Split Button is a hybrid between a Dropdown Menu and a Button. It lets users choose from
+  parallel actions and take action on their choice.
+package: '@zendeskgarden/react-buttons'
+components:
+  - buttons/src/elements/SplitButton.tsx
+  - buttons/src/elements/ChevronButton.tsx
+  - buttons/src/elements/Button.tsx
 ---
 
-Content here
+<Usage>
+  <Use>
+    <ul>
+      <li>To let users select from multiple parallel actions</li>
```

Comment:
@vsri1137 parallel actions feels kind of jargon-y. Can we define it for the users within this section, so they can be sure they understand?

Reply:
What do you think about: 

> To let users select from multiple parallel actions. Actions are parallel when each represents a path forward for the user and none cancel the action.

Reply:
@rossmoody let's get this integrated, and then i think we'll be ready for approval.